asset/cloud-client/quickstart_exportassets_test.py

```diff
@@ -18,19 +18,26 @@
 import time
 
 from google.cloud import storage
+from google.cloud import bigquery
 import pytest
 
 import quickstart_exportassets
 
 PROJECT = os.environ['GCLOUD_PROJECT']
 BUCKET = 'assets-{}'.format(int(time.time()))
```

Comment:
Can you use `uuid.uuid4()` instead of `int(time.time())`?  `time.time()` has second granularity. If multiple tests are running, the values easily conflict.